src/components/chips/js/contactChipsAutocompleteScope.js

```diff
@@ -0,0 +1,29 @@
+angular
+    .module('material.components.chips')
+    .directive('mdContactChipsAutocompleteScope', MdContactChipsAutocompleteScope);
```

Comment:
Can we rename this file to `contactChipsAutocompleteScopeDirective.js`?  It's verbose, but this is the convention we are using for the other files in here.